hadoop-ozone/dist/src/main/smoketest/env-compose.robot

```diff
@@ -13,4 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-org.apache.hadoop.fs.ozone.OzoneFileSystem
```

Comment:
Can you elaborate why this is removed/renamed? 

This seems to break the MR use case where the node manager running against o3fs got ClassNotFoundException on org.apache.hadoop.fs.ozone.OzoneFileSystem.

Reply:
Yes, this is a hard question. But as we have two different implementation (BasicOzoneFileSystem AND OzoneFileSystem) for the same scheme (o3fs) we can't use SPI only manual configuration.

With using META-INF based discovery it won't be possible to override the defined configuration.

Reply:
Make sense. +1 for this patch.